components/app.js

```diff
@@ -33,7 +37,33 @@ class App {
       error: this.handleGetGradesError
     });
   }
+  createGrade(name, course, grade) {
+    console.log(name, course, grade);
+
```

Comment:
Now that you've verified that creating a grade works, please remove this `console.log`.